### DISCUSSION TITLE
Heat/cool system

### DISCUSSION THREAD

**Discussion by highincolorado** [PeteRager/lennoxs30#56] [General]
Moved discussion to this project:

I do have a heat/cool system but the winter mode is heat only, on the thermostat card I am show a range instead of a single set point. 

Below the thermostat I am showing the local temperature from the outdoor sensor which s something their ios app does not show, very nice.

![FACC7B7A-AD55-4650-AB10-8BAE93DFC529](https://user-images.githubusercontent.com/60855018/137938873-93c61f25-81b4-49fc-8ee9-1018a2d8ad25.jpeg)
![13B7DD65-08E0-4D02-82E4-D5A6ACE46E60](https://user-images.githubusercontent.com/60855018/137938882-adcd3bd6-0436-4292-a675-e4695dd8898f.jpeg)

**Comment by PeteRager** [comment]
I see.   Your cool setpoint for that program is 85 degrees (CSP in the Lennox messages if you turn debug on) and your heat setpoint (HSP) is 71 degrees.  The system mode is Heat.  It is currently heating and running a schedule called Winter.

Since the system mode is heat, it is confusing to see the cool set point in the thermostat card and changing that cool setpoint won’t affect system operation anyways.   I agree that’s not ideal and we should try to fix it.   What does the S30 panel or Lennox Mobile app show?

What I don’t know, is if I can fix this in the integration or if the fix is in the thermostat card.

Let’s open an issue to track this item.

**Comment by highincolorado** [comment]
Yeah 85 must be the default when no cooling is called for. Attached in a screen shot of the ios app. I did flip between heat cool and heat/cool modes and it always showed both setpoints. Not as huge deal just mildly confusing. I have added the outdoor temperature and indoor humidity to my HA screen as well as a custom:mini-graph-card for the temperature. So far I haven't been able to figure out how to get the humidity into a custom:mini-graph-card. I imagine i'll have the same problem with the indoor temperature.

I'll fumble around and see about opening an issue.

Thanks

![Mainscreen](https://user-images.githubusercontent.com/60855018/137976544-2fb8b0fa-f8c3-40f4-b630-d238f34dd957.jpg
![HA screen](https://user-images.githubusercontent.com/60855018/137977517-5ac6864d-a8a1-49cf-ae2a-96960de0aa72.jpg)
)

**Comment by PeteRager** [comment]
Seems like there are some other alternatives to the stock climate card - lots of folks seem to use this one:

https://community.home-assistant.io/t/lovelace-simple-thermostat-card/82659/405

I don’t know If that will fix this issue.

To get the humidity attribute I created a sensor template and then historize that.   Same for temperature.   That seems to be what everyone does.   Could be a good enhancement to auto create these sensors just like outdoor temperature.

**Comment by highincolorado** [comment]
I'll give that card a try.
Seems I'll have to learn how to create a sensor template.
Need to fight off the squirrels and get to building my wifes new computer or there will be hell to pay.

More to come.

**Reply by PeteRager** [reply]
Something like this in configuration.yaml.  I can’t get it to preserve the idents :(


sensor:
   - platform: template   
     sensors:
       sagehouse_temperature:
           unit_of_measurement: F
           value_template: >- 
              {{ state_attr('climate.sagehouse_zone_1','current_temperature') }}
       sagehouse_humidity:
           unit_of_measurement: '%'
           value_template: >- 
              {{ state_attr('climate.sagehouse_zone_1','current_humidity') }}
       sagehouse_hvac_action:
           value_template: >- 
              {{ state_attr('climate.sagehouse_zone_1','hvac_action') }}
`

**Reply by highincolorado** [reply]
Ahhhh, new thing learned, thanks.

I have got to find that Home Assistant for dummys manual!

Thanks

**Comment by PeteRager** [comment]
We are going to automatically add the temperature and humidity sensors for each zone.  Tracking in this enhancement request

https://github.com/PeteRager/lennoxs30/issues/59
